tests/integration/Cron/SupervisorTest.php

```diff
@@ -21,51 +22,53 @@ function _before() {
     $this->settings->set('cron_trigger', [
       'method' => 'none',
     ]);
+    $this->cron_helper = ContainerWrapper::getInstance()->get(CronHelper::class);
+    $this->supervisor = ContainerWrapper::getInstance()->get(Supervisor::class);
```

Comment:
Just so you know, you can also use `$this->di_container->get(...)`. No need to change it now...